### DISCUSSION TITLE
Incorrect get farm AEP function?

### DISCUSSION THREAD

**Discussion by MYMahfouz** [NatLabRockies/floris#365] [Q&A]
The AEP calculation inside this function is not accurate

**Comment by rafmudaf** [comment]
Hi @MYMahfouz Thanks for the feedback. Can you elaborate?

**Reply by bayc** [reply]
Speculating here, but wondering if we should have a check that the frequency values passed by the user are the proper shape to get multiplied by the return of `get_farm_power()`.

**Reply by MYMahfouz** [reply]
Yes I believe this is one of the problems cause when I changed the frequency shape I got a different value.

**Comment by MYMahfouz** [comment]
I didn't look deeper into it to say exactly what goes wrong. It just sometimes gives different answer (much higher value for AEP) than what I get when I multiply the frequency with the power myself. I will try to have a deeper look and let you know what I think is wrong.
